### DISCUSSION TITLE
How to select a reference point in GMTSAR

### DISCUSSION THREAD

**Discussion by kjz1997** [gmtsar/gmtsar#833] [Q&A]
Hi professor
The results of insar may vary depending on the selected reference point, so I would like to know how to select a reference point when using gmtsar for processing

**Comment by Xiaohua-Eric-Xu** [comment]
Normally I would pick a location and set all unwrapped interferograms zero at that point. This can be done with `gmt grdmath/grdtrack` easily. However, it is recently found out that by doing so, not all motion are relative to the selected reference point. Stephenson, O. L., Liu, Y. K., Yunjun, Z., Simons, M., Rosen, P., & Xu, X. (2022). The Impact of Plate Motions on Long‐Wavelength InSAR‐Derived Velocity Fields. Geophysical Research Letters, 49(21), e2022GL099835.
